tasks/move-granules/index.js

```diff
@@ -136,6 +136,8 @@ async function moveFileRequest(file, sourceBucket, duplicateHandling) {
 
   const exists = await s3ObjectExists(target);
   log.debug(`file ${target.Key} exists in ${target.Bucket}: ${exists}`);
+  /* eslint-disable-next-line no-param-reassign */
+  if (exists) file.duplicate_found = true;
```

Comment:
Is the understanding that if no duplicate has been found, `file.duplicate_found` should be undefined? Looking at another [unit test](https://github.com/nasa/cumulus/blob/61eb268c787bfdeb2a60c89e6698d811513b9d84/packages/ingest/test/granule.js#L504), it looks like the answer is yes.

Reply:
Yes, that is the case. I'm not explicitly checking that in my unit test, though I could if you think it would be valuable

Reply:
I think it's fine as is. Just wanted to confirm that's the accepted idea.